lib/utils/logger/index.js

```diff
@@ -4,6 +4,7 @@ const {colors, colorsEnabled, disableColors} = require('../colors.js');
 
 const isErrorObject = require('../isErrorObject.js');
 const {errorToStackTrace} = require('../stackTrace.js');
+const lodashEscape = require('lodash/escape');
```

Comment:
this should be `lodash.escape` I think and you can also add a small test.

Reply:
any progress here?

Reply:
We don't have the entire `lodash` as a dependency. It's only as a devDependency. We only have these:
```
    "lodash.clone": "3.0.3",
    "lodash.defaultsdeep": "4.6.1",
    "lodash.merge": "4.6.2",
```

And of course we need to add `lodash.escape` as well.

Reply:
Added in #[5dc2064](https://github.com/nightwatchjs/nightwatch/pull/3270/commits/5dc2064b3d1419f950b7dfb090425f9b01dde360)